docs/docs/hooks.md

```diff
@@ -78,6 +78,9 @@ before_create:
 
 ### asg\_scheduled\_actions
 
+<div class="alert alert-warning">
+The asg_scheduled_actions hook has been deprecated, and will be removed in a later version of Sceptre. We recommend using the asg_scaling_processes hook instead.
```

Comment:
The comma after "deprecated" here (and in the warning message and tests) is a comma splice, I think.